js/misc/meson.build

```diff
@@ -5,7 +5,6 @@ jsconf.set('GETTEXT_PACKAGE', meson.project_name())
 jsconf.set('LIBMUTTER_API_VERSION', mutter_api_version)
 jsconf.set10('HAVE_BLUETOOTH', bt_dep.found())
 jsconf.set10('HAVE_NETWORKMANAGER', have_networkmanager)
-jsconf.set10('HAVE_PAYG', payg_dep.found())
```

Comment:
Please also remove the optional dependency itself. `payg_dep` is defined in the top-level `meson.build`.